
<!DOCTYPE html>
<html lang="en">


<head>
    <meta charset="UTF-8">
    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
    <title>MongoDB status</title>

    <link rel="stylesheet" href="https://slok.github.io/stactus-test/base/mongodb/css/main.css" />
</head>


<body>
    <div class="wrapper">
        <h1>MongoDB Cloud: Login error "Too many attempts, please try again later"</h1>
        <h2> Details </h2>
        <div>
            <table>
                <tr>
                    <td><strong>ID</strong></td>
                    <td>sjm3p6vmyq2m</td>
                </tr>
                <tr>
                    <td><strong>Severity</strong></td>
                    <td>Minor</td>
                </tr>
                <tr>
                    <td><strong>Started</strong></td>
                    <td>Sep  2, 12:13</td>
                </tr>
                <tr>
                    <td><strong>End</strong></td>
                    
                    <td>Sep  2, 16:28</td>
                    
                </tr>
                
                <tr>
                    <td><strong>Duration</strong></td>
                    <td>4h15m20.45s</td>
                </tr>
                
            </table>
        </div>
        <h2> Timeline </h2>
         
        <div class="box detail">
            <h3> Update - Sep 12, 16:36 </h3>
            <p> <h2 id="incident-summary">Incident Summary</h2>

<p>On September 2, 2024 between 8:30 UTC to 10:15 UTC, some MongoDB customers were unable to log in to MongoDB Atlas Management UI.</p>

<p>During that time, MongoDB experienced an abnormally high volume of traffic on our login page that caused us to hit our rate limiting thresholds which caused some customers to see their authentication requests to the MongoDB Atlas Management UI rejected.</p>

<p>Once login volume subsided, login availability was restored.</p>

<p>Note: This incident had no impact on customer Atlas Clusters. UI authentication is independent of Atlas Cluster authentication.</p>

<h2 id="root-cause">Root Cause</h2>

<p>Our authentication services were not able to scale quickly enough to react to the sudden increase in logins.</p>

<h2 id="mongodb-actions">MongoDB Actions</h2>

<p>We are updating our login page to provide clearer and more actionable error messaging. We are also adding additional support toÂ  scale better during traffic spikes.</p>

<h2 id="recommended-customer-actions">Recommended Customer Actions</h2>

<p>No further customer action required</p>
 </p>
        </div>
         
        <div class="box detail">
            <h3> Resolved - Sep  2, 16:28 </h3>
            <p> <p>This incident has been resolved.</p>
 </p>
        </div>
         
        <div class="box detail">
            <h3> Update - Sep  2, 13:08 </h3>
            <p> <p>The system has recovered, users should be able to login.</p>

<p>We are continuing to monitor the system for any residual effects.</p>
 </p>
        </div>
         
        <div class="box detail">
            <h3> Investigating - Sep  2, 12:13 </h3>
            <p> <p>We have received higher-than-normal reports of failed logins for cloud.mongodb.com with the reason &ldquo;Too many attempts, please try again later&rdquo;</p>
 </p>
        </div>
        

        <section>
            <a href="https://slok.github.io/stactus-test/base/mongodb/">Status</a>
        </section>
    </div>
    
<footer></footer>

</body>

</html>
